OpenSim/Simulation/Model/Appearance.h

```diff
@@ -33,105 +33,160 @@ namespace OpenSim {
-private:
-    void constructProperties() {
-        constructProperty_color(SimTK::Vec3(1.0));
-        constructProperty_opacity(1.0);
-        constructProperty_representation(3);
+    bool hasTexture() {
+        return !getProperty_texture_file().empty();
     }
-    //==========================================================================
-};  // END of class BaseAppearance
-
+    //========================================================================
+};  // END of class SurfaceAppearance
 
-class OSIMSIMULATION_API Appearance : public BaseAppearance {
-    OpenSim_DECLARE_CONCRETE_OBJECT(Appearance, BaseAppearance);
+//=============================================================================
+/** CurveAppearance class holds the Appearance properties of Geometry
+displayed in the OpenSim visualizer or GUI.The attributes in this
+class are specific to curves or line drawings so that thickness, line-style
+etc. can be maintained. 
+*/
+//=============================================================================
+class OSIMSIMULATION_API CurveAppearance : public Object {
+    OpenSim_DECLARE_CONCRETE_OBJECT(CurveAppearance, Object);
 public:
-    //==========================================================================
+    //=========================================================================
     // PROPERTIES
-    //==========================================================================
-    OpenSim_DECLARE_OPTIONAL_PROPERTY(texture_file, std::string,
-        "Name of file containing texture. ");
+    //=========================================================================
+    OpenSim_DECLARE_PROPERTY(thickness, double,
+        "The thickness of lines used to render a curve or a drawing. ");
 
     //--------------------------------------------------------------------------
     // CONSTRUCTION
     //--------------------------------------------------------------------------
 public:
-    Appearance() {
+    CurveAppearance() {
+        constructProperties();
     }
-    virtual ~Appearance() {};
+    virtual ~CurveAppearance() {};
 
-    bool hasTexture() {
-        return getProperty_texture_file().size() > 0;
+private:
+    void constructProperties() {
+        constructProperty_thickness(1.0);
     }
-//=============================================================================
-};  // END of class Appearance
-//=============================================================================
+    //=========================================================================
+};  // END of class CurveAppearance
 
 //=============================================================================
-class OSIMSIMULATION_API LineAppearance : public BaseAppearance {
-    OpenSim_DECLARE_CONCRETE_OBJECT(LineAppearance, Object);
+//=============================================================================
+/**
+ * A class that holds the Appearance properties of an object displayed 
+ * in the OpenSim Visualizer.
+ * 
```

Comment:
should say something about `Appearance` affects <i>how</i> Geometry is displayed.
It does not apply to Objects.